### PR TITLE
Add map of US States & Territories

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: packages/usa-states-territories/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+	"name": "@svg-maps/[new-map]",
```
Comment (VictorCazanave) [inline packages/usa-states-territories/package.json:2]: Could you replace the name here please?

File: packages/usa-states-territories/usa-states-territories.svg [hunk 2]
```diff
@@ -0,0 +1,287 @@
+<svg
+	xmlns="http://www.w3.org/2000/svg"
+	viewBox="0 0 928 587"
+	aria-label="Map of United States and Territories"
+>
+	<path
+		id="AK"
```
Comment (VictorCazanave) [inline packages/usa-states-territories/usa-states-territories.svg:7]: Could you use lowercase ids please?

File: packages/usa-states-territories/usa-states-territories.svg [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+	<path
+		id="VI"
+		name="Virgin Islands"
+		d="M646.4,524.1l-1.5,1.1l-1.8,1.1l0.4,1.5l1.8,0.7l2.6-0.4l1.1,1.8h1.8l1.1-1.1l4-0.7l1.8-0.4 l1.5-1.1l-0.7-1.5h-2.2l-5.1-1.1H646.4z M660.8,525.5l-0.3,1l0.9,0.5l2-0.2l1.4,0.7l1.1-0.2l0.3-1.2l0,0l1.4-1l0.3-1l-1.1-0.5 l-1.7,0.5l-2-0.7l-0.9,0.7l-0.3,1L660.8,525.5z M655.5,545.9l2.6-0.7l1.6,0.3l2.3-1.3l1,0.3l0.6,1l1,1.6l2-1l1,0.7l4.3-0.6l0.5,2.4 l-1.8,0.8h-3.3l-5.5,1.6l-2.3-0.3l-3.9,0.6l-1.3-0.6l-1,0.6l-2.6,0.3c0,0-1.5-0.5-0.3-1.3c2-1.3,1-3.6,1-3.6L655.5,545.9z">
+	</path>
+	<path
+		id="DC"
+		name="Washington, DC"
+		d="M801.8,253.8 l-1.1-1.6 -1-0.8 1.1-1.6 2.2,1.5z"/>
+	<circle id="DC2" stroke="#FFFFFF" stroke-width="1.5" cx="801.3" cy="251.8" r="5" opacity="1"/>
```
Comment (VictorCazanave) [inline packages/usa-states-territories/usa-states-territories.svg:285]: Could you delete this tag please?

**Review by VictorCazanave — COMMENTED** [review]
Thanks a lot for your contribution @fatima3558! 😃

Just few things to fix, then I'll glad to merge it.

**Comment by fatima3558** [comment]
had a little trouble there with `package-lock.json`, but I think I got it squared away. thanks for taking a look, let me know if any additional changes are needed.

**Comment by VictorCazanave** [comment]
Thanks again!

@all-contributors please add @fatima3558 for code

**Pull request merged**